lib/tbot/botfs/botfs.go

```diff
@@ -97,6 +99,29 @@ type ACLOptions struct {
 	ReaderUser *user.User
 }
 
+// ACLSelector is a target for an ACL entry, pointing at e.g. a single user or
+// group.
+type ACLSelector struct {
+	// User is a user specifier. If numeric, it is treated as a UID. Only one
+	// field may be specified per entry.
```

Comment:
> Only one field may be specified per entry.

This comment reads like should go on top for `ACLSelector`

Reply:
Ah, good catch, I've added a note to the struct itself, and repeated the warning for `Group` so the requirement is easily visible e.g. in IDE completions. I also added an actual call to `CheckAndSetDefaults()` since I just uh, outright forgot to call it 🤦